packages/billing/stripe-webhook-handlers/stripe.ts

```diff
@@ -0,0 +1,14 @@
+import Stripe from 'stripe';
+
+let stripe: Stripe;
+export function getStripe() {
```

Comment:
I need to create this function for test purpose, so I can mock the stripe instance in the test.